### PR TITLE
feat: documentation for data access consent get and post

### PR DESCRIPTION
**Pull request by Anouchka-M**
[UFSSP-39]

[UFSSP-39]: https://scalingo.atlassian.net/browse/UFSSP-39?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: _scalingo_api/data_access_consents.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+}
+```
+
+--- row ---
+
+## Create a New DataAccessConsent
+
+--- row ---
+
+`POST https://$SCALINGO_API_URL/v1/apps/:app_id/data_access_consents`
```
Comment (EtienneM) [inline _scalingo_api/data_access_consents.md:85]: nitpick: I find it surprising that `:app_id` is not the same as for the GET request. I would expect consistency here:
```suggestion
`POST https://$SCALINGO_API_URL/v1/apps/:app/data_access_consents`
```

File: _scalingo_api/data_access_consents.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+### Url Parameter
+
+- `example-app`: the urlized app name
```
Comment (EtienneM) [inline _scalingo_api/data_access_consents.md:89]:
```suggestion
- `app`: the urlized app name
```

File: _scalingo_api/data_access_consents.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+--- row ---
+
+## Create a New DataAccessConsent
+
+--- row ---
+
+`POST https://$SCALINGO_API_URL/v1/apps/:app_id/data_access_consents`
+
+### Url Parameter
```
Comment (EtienneM) [inline _scalingo_api/data_access_consents.md:87]:
```suggestion
### URL Parameter
```

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: _scalingo_api/data_access_consents.md (hunk 1 above)
Comment (Anouchka-M) [reply _scalingo_api/data_access_consents.md:85]: It's the get I forgot to edit correctly, with app_id !

File: _scalingo_api/data_access_consents.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+Example object:
+
+```json
+{
+  "data_access_consent": {
+    "id": "87b1ea4bfb0de6000fb8cbad",
+    "created_at": "2022-06-21T15:56:43.446Z",
+    "updated_at": "2022-06-21T15:56:43.446Z",
+    "app_id": "875ebfcbfb0de6000f7c9d93",
+    "user_id": "875941fa623d3a000f164df0",
```
[the remark this answers is not in the document]
Comment (Anouchka-M) [reply _scalingo_api/data_access_consents.md:35]: Ok, interesting.

File: _scalingo_api/data_access_consents.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+## Create a New DataAccessConsent
+
+--- row ---
+
+`POST https://$SCALINGO_API_URL/v1/apps/:app_id/data_access_consents`
+
+### URL Parameter
+
+- `app`: the urlized app name
```
Comment (EtienneM) [inline _scalingo_api/data_access_consents.md:89]:
```suggestion
- `app_id`: the app ID
```

**Review by EtienneM — APPROVED** [review]
Just a small change, but already LGTM

File: _scalingo_api/data_access_consents.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (btrd) [reply _scalingo_api/data_access_consents.md:35]: Do we have documentation about this ID vs UUID ?

[the remark this answers is not in the document]
Comment (btrd) [reply _scalingo_api/data_access_consents.md:35]: Leo respond on Slack #team-ufs

**Pull request merged**